lambdas/sessionstart/src/main/java/uk/gov/di/ipv/core/session/IpvSessionStartHandler.java

```diff
@@ -112,6 +114,21 @@ public APIGatewayProxyResponseEvent handleRequest(
             LOGGER.error("Failed to parse the decrypted JWE because: {}", e.getMessage());
             return ApiGatewayResponseGenerator.proxyJsonResponse(
                     HttpStatus.SC_BAD_REQUEST, ErrorResponse.INVALID_SESSION_REQUEST);
+        } catch (RecoverableJarValidationException e) {
+            LOGGER.error(
+                    "Recoverable Jar validation failed because: {}",
+                    e.getErrorObject().getDescription());
+
+            ClientSessionDetailsDto clientSessionDetailsDto =
+                    generateErrorClientSessionDetails(e.getRedirectUri(), e.getClientId());
+
+            String ipvSessionId =
+                    ipvSessionService.generateIpvSession(
+                            clientSessionDetailsDto, e.getErrorObject());
+
+            Map<String, String> response = Map.of(IPV_SESSION_ID_KEY, ipvSessionId);
+
+            return ApiGatewayResponseGenerator.proxyJsonResponse(HttpStatus.SC_OK, response);
```

Comment:
Small thing, could this catch go above the `ParseExeption` catch?  That way the `HttpStatus.SC_OK` responses are grouped at the top, and the bad ones at the bottom. Feel free to completely ignore this, it just made sense it my head.

Reply:
Sounds good I'll update it 👍